it/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -182,7 +185,54 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
       }
 
       "return a view which" should {
-        testCheckYourAnswersNoNinoView(result, testJourneyId)
+        testCheckYourAnswersNoNinoWithOverseasTaxIdentifierView(result, testJourneyId)
+      }
+
+      "redirect to sign in page" when {
+        "the user is UNAUTHORISED" in {
+          stubAuthFailure()
+          stubAudit()
+
+          lazy val result: WSResponse = get(s"/identify-your-sole-trader-business/$testJourneyId/check-your-answers-business")
+
+          result must have {
+            httpStatus(SEE_OTHER)
+            redirectUri("/bas-gateway/sign-in" +
+              s"?continue_url=%2Fidentify-your-sole-trader-business%2F$testJourneyId%2Fcheck-your-answers-business" +
+              "&origin=sole-trader-identification-frontend"
+            )
+          }
+
+          verifyAudit()
+        }
+      }
+    }
+
+    "the applicant does not have a nino and Overseas Tax Identifier details" should {
+      lazy val result: WSResponse = {
+        await(journeyConfigRepository.insertJourneyConfig(
+          journeyId = testJourneyId,
+          authInternalId = testInternalId,
+          journeyConfig = testSoleTraderJourneyConfig
+        ))
+        stubAuth(OK, successfulAuthResponse())
+        stubAudit()
+        stubRetrieveIndividualDetails(testJourneyId)(OK, testIndividualDetailsJsonNoNino)
+        stubRetrieveAddress(testJourneyId)(OK, testAddressJson)
+        stubRetrieveSaPostcode(testJourneyId)(OK, testSaPostcode)
+        stubRetrieveOverseasTaxIdentifier(testJourneyId)(NOT_FOUND)
+        stubRetrieveOverseasTaxIdentifierCountry(testJourneyId)(NOT_FOUND)
```

Comment:
You need to stub the old method here too, because if the new methods return not found the current methods will look for an overseas object as well.

Reply:
done